docs/design.html

```diff
@@ -508,7 +506,7 @@ <h4><a id="design_compactionbasics" href="#design_compactionbasics">Log Compacti
     the log, even if the message with that offset has been compacted away; in this case this position is indistinguishable from the next highest offset that does appear in the log. For example, in the picture above the
     offsets 36, 37, and 38 are all equivalent positions and a read beginning at any of these offsets would return a message set beginning with 38.
     <p>
-    Compaction also allows for deletes. A message with a key and a null payload will be treated as a delete from the log. Such a record is sometimes referred to as a <i>tombstone</i>. This delete marker will cause any prior message with that key to be removed (as would any new
+    Compaction also allows for deletes. A message with a key and a null payload will be treated as a delete from the log. This delete marker will cause any prior message with that key to be removed (as would any new
```

Comment:
Why was this sentence removed?

Reply:
I don't know, I missed this change when reviewing, I'll move it back